torch/nn/parallel/distributed.py

```diff
@@ -480,16 +476,12 @@ def __init__(
             os.environ.get("PYTORCH_DDP_USE_SIDE_STREAM", "1") == "1"
         )
 
+        # TODO(wayi@): Remove this field since SPMD is no longer supported,
+        # and also remove all the relevant unnecessary loops.
         # Module replication within process (single-process multi device)
-        self._module_copies = self._replicate_modules_within_process()
+        self._module_copies = [self.module]
         # Build parameters for reducer.
         parameters, expect_sparse_gradient = self._build_params_for_reducer()
-        # Verify model equivalence.
```

Comment:
Keep the relevant part of these comments (we still need to verify model equivalence across processes)?